e2e/rbd.go

```diff
@@ -498,6 +498,42 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and Bind it to an app with journaling/exclusive-lock image-features and rbd-nbd mounter",
+				func() {
+					err := deleteResource(rbdExamplePath + "storageclass.yaml")
+					if err != nil {
+						e2elog.Failf("failed to delete storageclass with error %v", err)
```

Comment:
`with error` is good to remove here @pkalever

Reply:
Thanks, as Niels already pointed, the issue doesn't seem to be from kernel driver as we expected. I will try to debug further, and refresh this later.